examples/benders_quatile_regression.jl

```diff
@@ -1,642 +1,642 @@
-#' ---
-#' title: Benders Decomposition for Norm-1 Regression with ParameterJuMP.jl
-#' author: Joaquim Dias Garcia
-#' date: March 9th 2019
-#' options:
-#'     github:
-#'         out_path : markdown
-#'     md2html:
-#'         out_path : html
-#'     md2pdf:
-#'         out_path : pdf
-#' ---
-
-#' # Introduction
-
-#' This notebook is parte of the talk on ParameterJuMP.jl in the
-#' third annual JuMP-dev workshop, held in Santiago, Chile, 2019
-
-#' The main purpose of this notebook is to show an application of
-#' [ParameterJuMP.jl](https://github.com/JuliaStochOpt/ParameterJuMP.jl).
-#' ParameterJuMP is well suited for Benders like decompositions
-#' therefore we shall try to demonstrate the usage of the library
-#' in one of the simplest problems that fits in the Benders
-#' decomposition framework. Norm-1 regression, which is a particular
-#' case of quantile regression is one of such problems.
-#'
-#' Note that this is NOT the standard technique to solve Norm-1
-#' regressions. Taylor made methods are available
-#' [here](https://cran.r-project.org/web/packages/quantreg/quantreg.pdf)
-#' for instance.
-#'
-
-#' This notebook will require the following libraries:
-
-#' ParameterJuMP itself
-using ParameterJuMP
-
-#' JuMP: the julia mathematical programming modeling tool
-using JuMP
-
-#' GLPK: A linear programing solver (other solvers could be used - 
-#' such as Clp, Xpress, Gurobi, CPLEX and so on)
-
-#+ echo = false
-# using Xpress
-# const OPTIMIZER = Xpress.Optimizer;
+#' ---


```

Comment:
What is your text editor ? Is is an issue between `\n` vs `\n\r` ?

Reply:
Vim. But I think the issue is rather with git's `autocrlf` params. Fixing that ASAP!